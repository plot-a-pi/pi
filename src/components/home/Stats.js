import React, { useState, useEffect } from 'react';
const Stats = () => {

  const [sampleSize, setSampleSize] = useState(0);
  const [piApproximation, setPiApproximation] = useState(0);

  useEffect(() => {
    //getSampleSize,
    //setSampleSize
    //getPiApproximaion,
    //setPiApproximation
  }, []);
  
  return (
    <>
      <h2>Sample Size:</h2>
      <p>{sampleSize}</p>
      <h2>Pi Approximation:</h2>
      <p>{piApproximation}</p>
    </>
  );
};

export default Stats;
