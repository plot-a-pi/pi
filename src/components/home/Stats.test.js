import React from 'react';
import Stats from './Stats';
import { shallow } from 'enzyme';
describe('Stats component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Stats />);
    expect(wrapper).toMatchSnapshot(); 
  });
})
;
