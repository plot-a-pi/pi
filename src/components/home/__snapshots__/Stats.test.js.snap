// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stats component should match snapshot 1`] = `
<Fragment>
  <h2>
    Sample Size:
  </h2>
  <p>
    0
  </p>
  <h2>
    Pi Approximation:
  </h2>
  <p>
    0
  </p>
</Fragment>
`;
